presto-orc/src/main/java/com/facebook/presto/orc/stream/ValueOutputStream.java

```diff
@@ -27,6 +27,10 @@
 
     StreamDataOutput getStreamDataOutput(int column);
 
+    /**
+     * When compression is enabled, this method returns
```

Comment:
This is try regardless of "compression".... maybe something more like "This method returns the size of the flushed data plus any unflushed data.  If the output is compressed, flush data size is the size after compression."